tests/core/nogo/custom/import_fmt_analyzer.go

```diff
@@ -0,0 +1,34 @@
+// importfmtanalyzer checks for the import of package fmt.
+package importfmtanalyzer
+
+import (
+	"go/ast"
+
+	"golang.org/x/tools/go/analysis"
+)
+
+const doc = `report imports of package fmt
+
+The import_fmt_analyzer analyzer reports imports of package fmt.`
+
+var Analyzer = &analysis.Analyzer{
+	Name: "import_fmt_analyzer",
+	Run:  run,
+	Doc:  doc,
+}
+
+func run(pass *analysis.Pass) (interface{}, error) {
+	for _, f := range pass.Files {
+		ast.Inspect(f, func(n ast.Node) bool {
+			switch n := n.(type) {
+			case *ast.ImportSpec:
+				if n.Path.Value == "\"fmt\"" {
```

Comment:
if path, _ := strconv.Unquote(n.Path.Value); path == "fmt" {

There are many string literals that denote the same string (e.g. \`fmt\`).

Reply:
You should never do that, of course, but an analysis tool should expect the unexpected.